src/runtime/composables/useDirectus.ts

```diff
@@ -9,13 +9,42 @@ import {
   graphql,
   rest
 } from '#imports'
+import defu from 'defu'
 
 export const useDirectus = <T extends Object>() => {
   return useNuxtApp().$directus as DirectusClient<T>
 }
 
 export const useDirectusRest = <T extends Object>(config?: RestConfig) => {
-  return useNuxtApp().$directus.with(rest(config)) as RestClient<T>
+  const { accessToken, refreshToken } = useDirectusCookie()
+  const staticToken = useRuntimeConfig().public.directus.staticToken
+
+  // TODO: add configs for oFetch once the following it's implemented
+  // https://github.com/directus/directus/issues/19592
+  const defaultConfig: RestConfig = {
+    // TODO: fix request for public content when accessToken is invalid.
+    // Current workaround check for refreshToken too to fallback to auto/manual refresh.
+    // @ts-ignore
```

Comment:
```suggestion
    // TODO!: Fix Types
    // @ts-ignore
```

Reply:
It was a type error on my machine not reading correctly `RestConfig` types